src/components/pages/InstructorHome/index.js

```diff
@@ -1,23 +1,59 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import '../../../styles/InstructorStyles/index.less';
 import InstructorSidebar from './InstructorSidebar';
-import CalendarApp from '../../common/calendar/Calendar';
 import { Layout } from 'antd';
+import CourseCard from '../AdminHome/AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { connect } from 'react-redux';
+//import AdminAddCoursesForm from '../AdminHome/AdminAddCoursesForm.js'
+//The above import will be used to add the add courses form, currently being developed in ticket BL-868
 
 const { Content } = Layout;
-const InstructorHome = () => {
+const InstructorHome = props => {
+  const { courses } = props;
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const idToken = useOktaAuth().oktaAuth.getIdToken();
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
+  }, [dispatch, idToken]);
+
+  const handleClick = () => {
+    history.push('/admin-add-course');
```

Comment:
Why is the push to admin-add-course? Would they select this route before hitting the instructor home path?